sdk/storage/azure-storage-file-share/azure/storage/fileshare/_version.py

```diff
@@ -4,4 +4,4 @@
 # license information.
 # --------------------------------------------------------------------------
 
-VERSION = "12.1.3"
+VERSION = "12.2.0"
```

Comment:
This should be a preview version.

Reply:
I'm really appreciated you've reviewed this pr!!!!! 
yeah I agree with you! when I had the pr, there wasn't a final decision made ^_^